### ISSUE TITLE
Chuck is not updated , What about Chucker . 

### ISSUE THREAD

**Issue by yudhir** [Commit451/Hyperion-Chuck#3]
https://github.com/ChuckerTeam/chucker

**Comment by Jawnnypoo** [comment]
Yeah, that is probably a good idea. Feel free to make a PR to change to this if you would like to see it done soon, I don't have much time right now to do this.

**Comment by yudhir** [comment]
Okay , how have you integrated it ?

**Comment by esabook** [comment]
I just fork this repo to use chucker, but not good for backward api is <21 and okHttp3
https://github.com/esabook/Hyperion-Chucker
